opensrp-app/src/test/java/org/ei/opensrp/commonregistry/CommonPersonObjectDateSortTest.java

```diff
@@ -0,0 +1,111 @@
+package org.ei.opensrp.commonregistry;
+
+import org.ei.opensrp.repository.AllBeneficiaries;
+import org.ei.opensrp.util.Cache;
+import org.ei.opensrp.view.contract.SmartRegisterClients;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.robolectric.RobolectricTestRunner;
+
+import java.util.Collections;
+import java.util.Map;
+
+import static java.util.Arrays.asList;
+import static junit.framework.Assert.assertEquals;
+import static org.ei.opensrp.util.EasyMap.create;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+/*
+ by Raihan Ahmed
+ */
+@RunWith(RobolectricTestRunner.class)
+public class CommonPersonObjectDateSortTest {
+    @Mock
+    private AllCommonsRepository allCommonsRepository;
+    @Mock
+    private AllBeneficiaries allBeneficiaries;
+
+    private CommonPersonObjectController controller;
+    private Map<String, String> emptyDetails;
+
+    private CommonObjectDateSort commonObjectdateSort;
+    @Before
+    public void setUp() throws Exception {
+        initMocks(this);
+        emptyDetails = Collections.emptyMap();
+        controller = new CommonPersonObjectController(allCommonsRepository, allBeneficiaries, new Cache<String>(), new Cache<CommonPersonObjectClients>(),"name","bindtype");
+    }
+
+    @Test
+    public void shouldSortCommonObjectsByColumnName() throws Exception {
+
+        commonObjectdateSort = new CommonObjectDateSort(CommonObjectDateSort.ByColumnAndByDetails.byColumn,"date");
+
+         Map<String, String> column1 = create("date", "2015-06-06").map();
+        Map<String, String> column2 = create("date","2015-07-07").map();
+        Map<String, String> column3= create("date","2015-01-01").map();
+
+
+
+
+        CommonPersonObjectClient expectedClient1 = new CommonPersonObjectClient("entity id 1",emptyDetails,"Woman A");
+        expectedClient1.setColumnmaps(column1);
+        CommonPersonObjectClient expectedClient2 = new CommonPersonObjectClient("entity id 2",emptyDetails,"Woman B");
+        expectedClient2.setColumnmaps(column2);
+        CommonPersonObjectClient expectedClient3 = new CommonPersonObjectClient("entity id 3",emptyDetails,"Woman C");
+        expectedClient3.setColumnmaps(column3);
+
+        SmartRegisterClients clients = new SmartRegisterClients();
+        clients.add(expectedClient1);
+        clients.add(expectedClient2);
+        clients.add(expectedClient3);
+
+
+//        Gson gson = new Gson();
```

Comment:
Could you remove these commented lines? It will makes the code appear less cluttered.